notifications/rest/notifications/send-notification-detailed/send-notification-detailed.php

```diff
@@ -0,0 +1,23 @@
+<?php
+// Get the PHP helper library from twilio.com/docs/php/install
+require './vendor/autoload.php';
+
+// Your Account Sid and Auth Token from twilio.com/user/account
+$sid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"; 
+$token = "{{ auth_token }}"; 
+
+// Initialize the client
+$client = new Twilio\Rest\Client($sid, $token);
+
+$service = $client->notifications->v1->services("ISxxx");
+
+$notification = $service->notifications->create(array(
```

Comment:
We use the PEAR style guide for php, and from [here](https://pear.php.net/manual/en/standards.funcalls.php), the way to indent this would be:

```
$notification = $service->notifications->create(
    array(
        identity => "Bob", 
        title => "Generic loooooooong title for all Bindings",
        body => "This is the body for all Bindings", 
        gcm => '{"notification":{"tag":"MyTag"}}', 
        apn => '{"aps":{"title":"Short title for Watch."}}'
    )
);
```